pkg/services/accesscontrol/resourceservices/resource_services.go

```diff
@@ -0,0 +1,93 @@
+package resourceservices
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/grafana/grafana/pkg/api/routing"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/services/accesscontrol/resourcepermissions"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+)
+
+func ProvideResourceServices(router routing.RouteRegister, sql *sqlstore.SQLStore, ac accesscontrol.AccessControl, store accesscontrol.ResourcePermissionsStore) (*ResourceServices, error) {
+	teamPermissions, err := ProvideTeamPermissions(router, sql, ac, store)
+	if err != nil {
+		return nil, err
+	}
+
+	return &ResourceServices{services: map[string]*resourcepermissions.Service{
+		"teams": teamPermissions,
+	}}, nil
+}
+
+type ResourceServices struct {
+	services map[string]*resourcepermissions.Service
+}
+
+func (s *ResourceServices) GetTeamService() *resourcepermissions.Service {
+	return s.services["teams"]
+}
+
+var (
+	TeamMemberActions = []string{
+		"teams:read",
+	}
+
+	TeamAdminActions = []string{
```

Comment:
As discussed in a group, we would like to add a permission for team admins to list roles mapped to that the team that they are an admin for. This can be done in a separate PR.